### PR TITLE
[NCL-473]: Rest get running build log returns 204 when it is empty

### PR DESCRIPTION
**Pull request by jbartece**

### CODE REVIEW

File: rest/src/main/java/org/jboss/pnc/rest/endpoint/RunningBuildRecordEndpoint.java [hunk 1]
```diff
@@ -66,10 +77,22 @@ public BuildRecordRest getSpecific(@ApiParam(value = "BuildRecord id", required
...
+            @ApiResponse(code = 200, message = "Successful retrieval of the log"),
+            @ApiResponse(code = 204, message = "RunningBuild exists, but the log content is empty"),
+            @ApiResponse(code = 404, message = "RunningBuild with specified id does not exist"),
+    })
     @ApiOperation(value = "Gets specific log of a Running Build Record")
     @GET
     @Path("/{id}/log")
-    public Response getLogs(@ApiParam(value = "BuildRecord id", required = true) @PathParam("id") Integer id) {
-        return Response.ok(buildRecordProvider.getLogsForRunningBuildId(id)).build();
+    public Response getLogs(@ApiParam(value = "RunningBuild id", required = true) @PathParam("id") Integer id) {
```
Comment (slaskawi) [inline rest/src/main/java/org/jboss/pnc/rest/endpoint/RunningBuildRecordEndpoint.java]: Similar situation here - shouldn't we just return null?

Comment (slaskawi) [inline rest/src/main/java/org/jboss/pnc/rest/endpoint/RunningBuildRecordEndpoint.java]: ooh yes I see - we also have 404 for not found. Please ignore my previous comment.

**Comment by slaskawi** [comment]
Looks good to me :+1:

**Comment by jbartece** [comment]
retest this please

**Pull request merged**